application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -881,6 +887,32 @@ type LocalTimeSpanDate {
   timeSpans: [LocalTimeSpan]
 }
 
+"""
+A stop that isn't a fixed point but zone where passengers can board or alight anywhere.
+
+This is mostly used by demand-responsive services.
+"""
+type Location {
+  "The geometry representing the geographic extend of the location."
+  geometry: StopGeometries!
+  gtfsId: String!
+  "Optional name of the location."
+  name: String
+}
+
+"""
+A group of fixed stops that is visited in an arbitrary order.
+
+This is mostly used by demand-responsive services.
+"""
+type LocationGroup {
+  gtfsId: String!
+  "The members that are part of the group."
+  members: [Stop!]!
```

Comment:
Btw can these stops be stations? I think we should document it here regardless of how it is.